consumers/upstream.go

```diff
@@ -0,0 +1,234 @@
+package consumers
+
+import (
+	"fmt"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/flanksource/duty/upstream"
+	"github.com/flanksource/postq"
+)
+
+const EventPushQueueCreate = "push_queue.create"
```

Comment:
I think this should go into duty, I want to try and make postq a general library that we can publicise - which means we should not import duty or commons